storage-proofs/src/piece_inclusion_proof.rs

```diff
@@ -90,10 +133,44 @@ fn generate_piece_commitment<H: Hasher>(data: &[u8]) -> Result<H::Domain> {
 
 #[allow(dead_code)]
 /// Generate `comm_p` from bytes and return it as bytes.
-fn generate_piece_commitment_bytes<H: Hasher>(data: &[u8]) -> Result<Fr32Vec> {
+fn generate_piece_commitment_bytes<H: Hasher>(data: &[u8]) -> Result<Fr32Ary> {
```

Comment:
TODO: `generate_piece_commitment_bytes` is a test-only function and should be moved into a `tests` module.